x/mongo/driver/errors.go

```diff
@@ -316,15 +318,16 @@ func (e Error) NodeIsShuttingDown() bool {
 	return hasNoCode && strings.Contains(e.Message, "node is shutting down")
 }
 
-// NotMaster returns true if this error is a not master error.
-func (e Error) NotMaster() bool {
-	for _, code := range notMasterCodes {
+// NotPrimary returns true if this error is a not primary error.
+func (e Error) NotPrimary() bool {
+	for _, code := range notPrimaryCodes {
 		if e.Code == code {
 			return true
 		}
 	}
 	hasNoCode := e.Code == 0
-	return hasNoCode && strings.Contains(e.Message, "not master")
+	return hasNoCode &&
+		(strings.Contains(e.Message, internal.LegacyNotPrimary) || strings.Contains(e.Message, "not primary"))
```

Comment:
Similarly, I think this can remove the "not primary" check.

Reply:
Removed.